ditto-markov/docker-compose.yaml

```diff
@@ -11,4 +11,29 @@ services:
       - ./ditto_markov:/app/ditto_markov
       - ./poetry.lock:/app/poetry.lock
       - ./pyproject.toml:/app/pyproject.toml
-    entrypoint: ["poetry", "run", "cli",]
\ No newline at end of file
+    entrypoint: ["poetry", "run", "cli",]
+
+  cosmosdb:
+    image: ridhoq/cosmosdb-server:0.7.0
+    command: ["-p", "3000"]
+    ports:
+    - 3000
+    
+  test:
+    depends_on:
+      - cosmosdb
+    build:
+      context: .
+      target: development
+    image: ridhoq/ditto/ditto-markov/test
+    environment:
+      COSMOS_ACCOUNT_URI: https://cosmosdb:3000
+      COSMOS_DATABASE_ID: ditto-test
+      COSMOS_DISABLE_TLS: "True"
+    volumes:
+      - ./pytest.ini:/app/pytest.ini
```

Comment:
what was this mount for?

Reply:
I was tryna get warning suppression to work but we don't really need it to be mounted. I'll remove that in the next PR 👍